CMake/AbseilDll.cmake

```diff
@@ -349,121 +349,142 @@ set(ABSL_INTERNAL_DLL_FILES
 )
 
 set(ABSL_INTERNAL_DLL_TARGETS
-  "stacktrace"
-  "symbolize"
-  "examine_stack"
-  "failure_signal_handler"
-  "debugging_internal"
-  "demangle_internal"
-  "leak_check"
-  "leak_check_disable"
-  "stack_consumption"
-  "debugging"
-  "hash"
-  "spy_hash_state"
-  "city"
-  "memory"
-  "strings"
-  "strings_internal"
-  "cord"
-  "str_format"
-  "str_format_internal"
-  "pow10_helper"
-  "int128"
-  "numeric"
-  "utility"
+  "algorithm"
+  "algorithm_container"
   "any"
+  "atomic_hook"
   "bad_any_cast"
   "bad_any_cast_impl"
-  "span"
-  "optional"
   "bad_optional_access"
   "bad_variant_access"
-  "variant"
-  "compare"
-  "algorithm"
-  "algorithm_container"
-  "graphcycles_internal"
-  "kernel_timeout_internal"
-  "synchronization"
-  "thread_pool"
+  "base"
+  "base_internal"
   "bind_front"
-  "function_ref"
-  "atomic_hook"
-  "log_severity"
-  "raw_logging_internal"
-  "spinlock_wait"
+  "bits"
+  "btree"
+  "city"
+  "civil_time"
+  "compare"
+  "compressed_tuple"
   "config"
-  "dynamic_annotations"
+  "container"
+  "container_common"
+  "container_memory"
+  "cord"
+  "cord_internal"
+  "cordz_functions"
+  "cordz_handle"
+  "cordz_info"
+  "cordz_sample_token"
   "core_headers"
-  "malloc_internal"
-  "base_internal"
-  "base"
-  "throw_delegate"
-  "pretty_function"
+  "counting_allocator"
+  "debugging"
+  "debugging_internal"
+  "demangle_internal"
+  "dynamic_annotations"
   "endian"
-  "bits"
+  "examine_stack"
   "exponential_biased"
-  "periodic_sampler"
-  "scoped_set_env"
-  "type_traits"
+  "failure_signal_handler"
+  "fixed_array"
+  "flags"
```

Comment:
Because of the nature of the flags library (and its dependencies), it was intentionally left out of the DLL build, and we consider flags unsupported for this use case. Can you remove the flags related targets?

We have a policy of not allowing any non-flags targets to depend on flags, so this doesn't change the usability of anything else.